src/mesh_lib/node/packet/trait_implementations/serializer.rs

```diff
@@ -95,11 +93,11 @@ impl Serializer for Packet {
         let mut data: PacketDataBytes = PacketDataBytes::new();
         for _ in 0..DATA_TYPE_SIZE {
             data.push(*bytes_iterator.next().unwrap_or_else(|| {
-                serial_debug!("Could not take byte for deserialization of data");
+                // serial_debug!("Could not take byte for deserialization of data");
                 &0u8
             }))
             .unwrap_or_else(|_| {
-                serial_debug!("Could not push byte of serialized data");
+                // serial_debug!("Could not push byte of serialized data");
```

Comment:
Can such macrosses be defined in platform-serial crate, with no need to re-define them each time in any crate like platform-serial-arduino-nano, etc??

Reply:
use standard `log` macros? info! debug! etc

Reply:
@berkus If i just regular user, and I do have only one physical serial port on the board, how could i use this library and log? Like share one serial stream to debug and transfer data? Thankfully - the protocol is pretty stable to handle that case.